server/src/test/java/io/crate/integrationtests/UpdateIntegrationTest.java

```diff
@@ -1132,4 +1127,17 @@ public void test_update_on_table_with_big_refresh_interval_emptied_by_delete_ret
         execute("update test set a = 2 where a = 1");
         assertThat(response).hasRowCount(0L);
     }
+
+    @Test
+    public void test_update_continues_on_error_if_at_least_1_row_was_updated() {
+        execute("create table test (a int CHECK (a < 100))");
+
+        execute("insert into test (a) values (1), (2), (3)");
+        assertThat(response).hasRowCount(3);
+        refresh();
+
+        // Trying to update 1, 2, 3 to (1 + 100), (2 + 0), (3 + 100), only (2 + 0) doesn't violate CHECK.
+        execute("update test set a = a + 100 * (a % 2)");
```

Comment:
Wouldn't this be much simpler?
```suggestion
        execute("update test set a = a + 98");
```